src/platform-includes/performance/configure-sample-rate/javascript.vue.mdx

```diff
@@ -57,3 +53,5 @@ Sentry.init({
   hooks: ["create", "mount"],
 });
 ```
+
+Learn more about [Vue Component Tracking](../features/componenttracking).
```

Comment:
Since we're saying _learn about subject x_ vs _learn about subject x in doc x_, I've made it lower case like in the previous include. 

```suggestion
Learn more about [Vue component tracking](../features/componenttracking).
```